src/pages/post-test-summary/post-test-summary.ts

```diff
@@ -25,29 +23,32 @@ export class PostTestSummaryPage {
   seriousFaultSummary: IFaultSummary;
   dangerousFaultSummary: IFaultSummary;
   journalPage: Page = JournalPage;
-  selectedRoute: number = null;
+  selectedRoute: string = null;
```

Comment:
I notice the route number selection modal is gone in favour of keyboard based entry. Is this the desired behaviour?

Reply:
Yes - The Route Numbers go from 1 - 99 having a modal/popover/select wasn't working and PO's agreed that we should change it to a text box and limit it to 2 characters